great_expectations/cli/batch_request.py

```diff
@@ -250,6 +263,15 @@ def _get_default_schema(datasource: SimpleSqlalchemyDatasource) -> str:
     return inspector.default_schema_name
 
 
+def _check_default_data_connectors(available_data_asset_names_by_data_connector_dict):
+    if all(
+        data_connector_name in available_data_asset_names_by_data_connector_dict
+        for data_connector_name in DEFAULT_DATA_CONNECTOR_NAMES
+    ):
+        # return the default_inferred_asset_data_connector
+        return DEFAULT_DATA_CONNECTOR_NAMES[1]
+
+
```

Comment:
Can we add type hints? Just to make the expected behavior very clear 😬 
```suggestion
def _check_default_data_connectors(available_data_asset_names_by_data_connector_dict: Dict[str, List[str]]) -> Optional[str]:
    if all(
        data_connector_name in available_data_asset_names_by_data_connector_dict
        for data_connector_name in DEFAULT_DATA_CONNECTOR_NAMES
    ):
        # return the default_inferred_asset_data_connector
        return DEFAULT_DATA_CONNECTOR_NAMES[1]


```